### PR TITLE
Refactor workers ai docs to use native binding instead of npm package

### PR DESCRIPTION
**Pull request by G4brym**

### CODE REVIEW

**Review by markdembo — APPROVED** [review]
LGTM - can we please announce the deprecation here too:
https://developers.cloudflare.com/changelog/

I am not sure how that works

File: data/changelogs/workers-ai.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+link: "/radar/changelog/"
+productName: Workers AI
+entries:
+- publish_date: '2024-04-11'
+  title: Add AI native binding
+  description: |-
+    * Added new AI native binding, you can now run models with `const resp = await env.AI.run(modelName, inputs)`
+    * Deprecated `@cloudflare/ai` npm package, customers can continue using it, but its recommended that they change to the new native binding
```
Comment (markdembo) [inline data/changelogs/workers-ai.yaml]: Proposal:     *  Deprecated @cloudflare/ai npm package. While existing solutions using the @cloudflare/ai package will continue to work, no new Workers AI features will be supported. Moving to native AI bindings is highly recommended.

File: content/workers-ai/configuration/bindings.md [hunk 2]
```diff
@@ -30,4 +30,43 @@ binding = "AI" # i.e. available in your Worker on env.AI
...
+
+
+## Methods
+
+### async env.AI.run()
+
+`async env.AI.run()` runs a model. Takes a model as the first parameter, and an object as the second parameter.
+
+```javascript
+const answer = env.AI.run('@cf/meta/llama-2-7b-chat-int8', {
```
Comment (WalshyDev) [inline content/workers-ai/configuration/bindings.md]:
```suggestion
const answer = await env.AI.run('@cf/meta/llama-2-7b-chat-int8', {
```

**Review by WalshyDev — COMMENTED** [review]
Looks good from Pages side, Ty

File: data/changelogs/workers-ai.yaml (hunk 1 above)
Comment (G4brym) [reply data/changelogs/workers-ai.yaml]: Thanks, text updated

File: data/changelogs/workers-ai.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+link: "/radar/changelog/"
+productName: Workers AI
+entries:
+- publish_date: '2024-04-11'
+  title: Add AI native binding
+  description: |-
+    * Added new AI native binding, you can now run models with `const resp = await env.AI.run(modelName, inputs)`
+    * Deprecated `@cloudflare/ai` npm package. While existing solutions using the @cloudflare/ai package will continue to work, no new Workers AI features will be supported.
+      Moving to native AI bindings is highly recommended
```
Comment (kodster28) [inline data/changelogs/workers-ai.yaml:10]:
```suggestion
      Moving to native AI bindings is highly recommended
```
```suggestion
     * Moving to native AI bindings is highly recommended
```

**Review by craigsdennis — APPROVED** [review]
Nice work Gabriel!

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**